### PR TITLE
Font updates

### LINKED ISSUE #799: Font updates
As per design re-review with Jimmy

### PR DESCRIPTION
**Pull request by AndyEPhipps**
fixes #799

### CODE REVIEW

File: sass/themes/cr/2019/variables/_typography.scss [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
 );
 $h1-font-size: (
-  null: (64px, 72px),
+  null: (42px, 48px),
   $screen-md: (64px, 72px),
   $screen-lg: (64px, 72px)
 );
 $h2-font-size: (
-  null: (42px, 48px),
+  null: (34px, 40px),
```
Comment (Heleen-cr) [inline sass/themes/cr/2019/variables/_typography.scss]: Check against twig file. Is it 32x40 or 34x40?

File: sass/themes/cr/2019/components/typography/typography.twig [hunk 2]
```diff
@@ -26,23 +26,23 @@
...
+ <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 42px / line-height: 48px</span>
 <br/>
 <br/>
 <br/>
 
 <h2>Header 2</h2>
   <span class="breakpoint-lg font--red"><b>Current breakpoint: LG </b> - font-size: 42px / line-height: 48px</span>
   <span class="breakpoint-md font--red"><b>Current breakpoint: MD </b> - font-size: 42px / line-height: 48px</span>
-  <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 42px / line-height: 48px</span>
+  <span class="breakpoint-sm font--red"><b>Current breakpoint: SM </b> - font-size: 32px / line-height: 40px</span>
```
Comment (Heleen-cr) [inline sass/themes/cr/2019/components/typography/typography.twig]: Check with font mixin. Is size 32 or 34 px?

File: sass/themes/cr/2019/variables/_typography.scss (hunk 1 above)
Comment (AndyEPhipps) [reply sass/themes/cr/2019/variables/_typography.scss]: Whoops. SASS right, Twig wrong; have corrected

File: sass/themes/cr/2019/components/typography/typography.twig (hunk 2 above)
Comment (AndyEPhipps) [reply sass/themes/cr/2019/components/typography/typography.twig]: clip-art-number-34-clipart-1

**Pull request merged**